packages/core/saved-objects/core-saved-objects-api-server-internal/src/lib/apis/helpers/index.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { ICommonHelper } from './common';
+import type { IEncryptionHelper } from './encryption';
+import type { IValidationHelper } from './validation';
+import type { IPreflightCheckHelper } from './preflight_check';
+import type { ISerializerHelper } from './serializer';
```

Comment:
Interesting, how did this manifest as an issue? Were you able to access private members in the repository functions?

Reply:
TS complained because the mocked version (that only implements the methods) could not be coerced as the concrete type because it was missing the private properties